modules/Data_Visualization/Data_Visualization.Rmd

```diff
@@ -416,19 +416,32 @@ It's possible to go in and change specifications with newer layers
 
 ```{r, fig.width=5, fig.height=3, fig.align='center'}
 Orange %>% ggplot(aes(x = circumference,
-                      y = age, 
-                      color = Tree)) + 
+                      y = age,
+                      color = Tree)) +
   geom_line(size = 0.8)
 ```
 
+## Removing the legend label
```

Comment:
This is a change (there's weird spacing that showed up in the diff so I'm just going to point out the "real" changes).

Reply:
thank you!